src/hooks/screen/useChatScreenHook.ts

```diff
@@ -208,29 +227,21 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           );
 
           await currentChatSchema?.save(localDb);
-          initChatData();
+          handleChatData();
         } else {
-          queue.addPriorityJob({
-            operationLabel: DatabaseOperationLabel.ChatScreen_SendChat,
-            id: `${selectedChannel?.id}-${new Date().valueOf()}`,
-            priority: QueueJobPriority.HIGH,
-            forceAddToQueue: true,
-            task: async () => {
-              currentChatSchema = await ChatSchema.generateSendingChat(
-                randomId,
-                userId,
-                selectedChannel?.id || '',
-                message,
-                attachments,
-                localDb,
-                'regular',
-                'pending'
-              );
+          currentChatSchema = await ChatSchema.generateSendingChat(
+            randomId,
+            userId,
+            selectedChannel?.id || '',
+            message,
+            attachments,
+            localDb,
+            'regular',
+            'pending'
+          );
 
-              await currentChatSchema?.save(localDb);
-              initChatData();
-            }
-          });
+          await currentChatSchema?.save(localDb);
+          handleChatData();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleChatData` function is called twice in this code block, once for each branch of the if-else statement. To avoid repetition and improve maintainability, consider calling this function once after the if-else statement.
```diff
  if (isAnonymous) {
    currentChatSchema = await ChatSchema.generateSendingChat(
      randomId,
      userId,
      selectedChannel?.id || '',
      message,
      attachments,
      localDb,
      'anonymous',
      'pending'
    );

    await currentChatSchema?.save(localDb);
-   handleChatData();
  } else {
    currentChatSchema = await ChatSchema.generateSendingChat(
      randomId,
      userId,
      selectedChannel?.id || '',
      message,
      attachments,
      localDb,
      'regular',
      'pending'
    );

    await currentChatSchema?.save(localDb);
-   handleChatData();
  }

+ handleChatData();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->